tests/test_memory_ingest_workflow_check.py

```
@@ -0,0 +1,444 @@
+        return node
+
+    def _node(self, name: str, node_type: str, parameters: Optional[dict] = None) -> dict:
+        return {
+            "name": name,
+            "type": node_type,
+            "parameters": parameters or {},
+        }
+
+    def _run_check(self, repo_root: Path) -> subprocess.CompletedProcess[str]:
+        return subprocess.run(
+            ["bash", "scripts/ci/memory_ingest_workflow_check.sh"],
```

Ruff (0.15.9)

[error] Starting a process with a partial executable path

(S607)